types/selenium-webdriver/lib/virtual_authenticator.d.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Protocol for virtual authenticators
+ * @enum {string}
+ */
+export interface IProtocol {
+    CTAP2: string;
+    U2F: string;
+}
+
+/**
+ * AuthenticatorTransport values
+ * @enum {string}
+ */
+export interface ITransport {
+    BLE: string;
+    USB: string;
+    NFC: string;
+    INTERNAL: string;
+}
+
+export const Protocol: IProtocol;
+export const Transport: ITransport;
+
+/**
+ * Options for the creation of virtual authenticators.
+ * @see http://w3c.github.io/webauthn/#sctn-automation
+ */
+export class VirtualAuthenticatorOptions {
+    constructor();
+
+    getProtocol(): string;
+
+    setProtocol(protocol: string): void;
+
+    getTransport(): string;
+
+    setTransport(transport: string): void;
```

Comment:
```suggestion
    getProtocol(): Protocol;

    setProtocol(protocol: Protocol): void;

    getTransport(): Transport;

    setTransport(transport: Transport): void;
```

Making those two consts into enums means they can be used as types here

Reply:
Change incorporated. Thank you @SupernaviX